README.md

```diff
@@ -51,6 +51,8 @@ Here are the steps for provisioning a new cluster:
         -e "CLUSTER_NAME=$CLUSTER_NAME" \
         -e "CLUSTER_ENVIRONMENT=$CLUSTER_ENVIRONMENT" \
         -e "ENVIRONMENT_TYPE=$ENVIRONMENT_TYPE" \
+        -e "KONSTRUCTOR_API_KEY=$KONSTRUCTOR_API_KEY" \
```

Comment:
Those being as params will complicate a little bit the Jenkins job that updates the cluster automatically. Can we keep at least the Konstructor key in vault?

Reply:
On this note, this should not be merged in master until we also update the Jenkins job

Reply:
Updated the lastpass entry 👍